test/phonons.jl

```diff
@@ -4,6 +4,22 @@ using DFTK
 using LinearAlgebra
 using ForwardDiff
 using StaticArrays
+using Random
+
+# Gaussian local potential
+struct ElementGaussian <: DFTK.Element
+    α  # Prefactor
+    L  # Width of the Gaussian nucleus
+    symbol
+end
+
+function DFTK.local_potential_real(el::ElementGaussian, r)
+    -el.α / (√(2π) * el.L) * exp(- (r / el.L)^2 / 2)
+end
+function DFTK.local_potential_fourier(el::ElementGaussian, q::Real)
+    ## = ∫ V(x) exp(-ix⋅q) dx
+    -el.α * exp(- (q * el.L)^2 / 2)
+end
```

Comment:
OK, we use this element gaussian quite a lot. Can you maybe put it in src/elements.jl (in a separate PR)?